ui/app/pages/confirm-transaction-base/confirm-transaction-base.container.js

```diff
@@ -172,7 +180,13 @@ const mapDispatchToProps = dispatch => {
     },
     cancelTransaction: ({ id }) => dispatch(cancelTx({ id })),
     cancelAllTransactions: (txList) => dispatch(cancelTxs(txList)),
-    sendTransaction: txData => dispatch(updateAndApproveTx(txData)),
+    sendTransaction: txData => {
+      const _txData = customNonceValue ? {
+        ...txData,
+        customNonceValue,
```

Comment:
I tried doing: `nonce: customNonceValue` to see if i could get the nonce into the transaction that way, but it didn't appear to work as expected...

Reply:
This is due to how the nonce is set when a transaction is approved. See line 379 of the transactions controller: https://github.com/MetaMask/metamask-extension/pull/7089/files/5e653e49f9376c12a33b77ce5813bb30468c2c33#diff-97c997fde8231c768b7f2393fd3012e1L379

It looks like you were on your way to updating that code based on your comment in that file